webapp/src/widgets/editable.tsx

```diff
@@ -116,23 +116,14 @@ export function useEditable(
     }
 }
 
-function borderWidth(style: CSSStyleDeclaration): number {
-    return (
-        parseInt(style.borderLeftWidth || '0', 10) +
-        parseInt(style.borderRightWidth || '0', 10)
-    )
-}
-
 const Editable = (props: EditableProps, ref: React.Ref<Focusable>): JSX.Element => {
     const elementRef = useRef<HTMLInputElement>(null)
     const elementProps = useEditable(props, ref, elementRef)
 
     useLayoutEffect(() => {
         if (props.autoExpand && elementRef.current) {
             const input = elementRef.current
-            const computed = getComputedStyle(input)
-            input.style.width = 'auto'
-            input.style.width = `${input.scrollWidth + borderWidth(computed) + 1}px`
+            input.style.width = '100%'
```

Comment:
@redhoyasa @sbishel this is really strange change: the whole point of using layout effect was that we need to compute the required width for input element. Just setting width to `100%` can be easily done via CSS.

Also this change leads to regression in the card dialog because the text properties including URL now have very wide input element:
<img width="841" alt="image" src="https://user-images.githubusercontent.com/5587620/200823861-be8786ad-cf2a-444b-b2d4-6aeeb9620a51.png">